.gitignore

```diff
@@ -20,4 +20,5 @@ yarn-error.log*
 # package directories
 jspm_packages
 # Serverless directories
-.serverless
\ No newline at end of file
+.serverless
+/oss-cloud-backend/.env
```

Comment:
Let's add separate .gitignore files inside frontend and backend project directories and list project specific files/dirs there.

Reply:
They were separate, but we were told to but them together. I think both options are viable.